third_party/2and3/redis/client.pyi

```diff
@@ -121,7 +123,7 @@ class Redis(object):
     __contains__: Any
     def expire(self, name: Union[Text, bytes], time: Union[int, timedelta]) -> bool: ...
     def expireat(self, name, when): ...
-    def get(self, name: Union[Text, bytes]) -> Optional[bytes]: ...
+    def get(self, name: Union[Text, bytes]): ...
```

Comment:
Why did you remove the return type here?

Reply:
sorry, i should have explained.

because it can return either a bytes or a string, depending on if the redis client class was initialized with `decode_repsonses=True`. from a typing perspective this is a bit of a problematic class, maybe it could be generic and overload the operator.

I decided the best solution for now is to not specify the return type at all, which is consistent with almost all of the other methods in the class.

Reply:
Per my explanation above, we should use `Any` here then. We also often add a comment with the "real" return type in those cases as an additional hint to readers.